conf/test.config

```diff
@@ -38,4 +38,7 @@ params {
     // Polishing options
     enable_polishing       = false
     polishing_n_chunks_bed = 2
+
+    // Hi-C mapping options
+    hic_mapping_cram_chunk_size = 100
```

Comment:
is this container size? means 100 containers/chunk ?

Reply:
Yes

Reply:
I was trying to make the test spawn multiple chunks, but not too many - settled on 100 as a good number for that, it results in about 8